src/controllers/tracking.controller.ts

```
@@ -0,0 +1,19 @@
+
+export class TrackingController {
+  constructor(private trackingService: TrackingService) {}
+
+  track = (async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      const { type } = req.body;
+      const { id } = req.user;
+      const result = await this.trackingService.tracking(type, id);
+      return res.status(200).json({ success: true, message: '저장완료', data: result });
+      next();
+    } catch (error) {
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)